etcdserver/membership/cluster.go

```diff
@@ -307,21 +307,21 @@ func (c *RaftCluster) RemoveMember(id types.ID) {
 	c.removed[id] = true
 }
 
-func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) bool {
+func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) {
```

Comment:
@heyitsanthony Sorry. I missed your last comment. It does not matter since no one consumes the response: https://github.com/coreos/etcd/blob/master/etcdserver/server.go#L924. I think I should add a comment to make it clear.